test/e2e/move.update.sch.test.js

```diff
@@ -47,10 +46,6 @@ test.meta('hasDocument', 'true')(
   }
 )
 
-test('User should be able to cancel move', async () => {
```

Comment:
These were added in by me, as this functionality was specified in the original story - but if they're unduly slowing down the tests, they're probably one of the safest tests to remoe.